public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -43,7 +43,7 @@ interface PolicyComponentProps {
   contactPointsState?: ReceiversState;
   readOnly?: boolean;
   inheritedProperties?: InhertitableProperties;
-  routesMatchingFilters?: RouteWithID[];
+  routesMatchingFilters?: string[];
```

Comment:
why we don't need this recursive type anymore?

Reply:
I think this is outdated version. In the latest one there is no change in this line